bedrock/firefox/templates/firefox/includes/family-nav.html

```diff
@@ -0,0 +1,153 @@
+{#
+** OFFICIAL™ DOCS **
```

Comment:
Would these awesomely official docs be better in the actual `/docs`? :trollface: